### ISSUE TITLE
python for json

### ISSUE THREAD

**Issue by kiss2013** [kiss2013/cbio#86]
import json
import numpy as np

def validate_model_against_experiment(json_file='experimental_data.json'):
    with open(json_file, 'r') as f:
        data = json.load(f)
    
    R = data['reference_metadata']['ball_params']['radius']
    mu = 0.20  # Базовый коэф. трения
    ey = 0.90  # Коэф. восстановления
    
    print(f"{'Test Case':<25} | {'Exp. Loss %':<12} | {'Model Loss %':<12} | {'Error %'}")
    print("-" * 75)

    for case in data['test_cases']:
        omega_0 = case['initial']['omega']
        vy_0 = abs(case['initial']['vy'])
        exp_omega_1 = case['measured_after_bounce']['omega']
        
        # Расчет по модели Дюре-Сейделя
        # Delta_omega = (3*mu / 2*R) * (1 + ey) * Vy
        delta_omega_rad = (3 * mu / (2 * R)) * (1 + ey) * vy_0
        delta_omega_rps = delta_omega_rad / (2 * np.pi)
        model_omega_1 = omega_0 - delta_omega_rps
        
        # Расчет погрешностей
        exp_loss = (1 - exp_omega_1 / omega_0) * 100
        model_loss = (1 - model_omega_1 / omega_0) * 100
        error = abs(exp_loss - model_loss)
        
        print(f"{case['label']:<25} | {exp_loss:>11.1f}% | {model_loss:>11.1f}% | {error:>6.1f}%")

if __name__ == "__main__":
    validate_model_against_experiment()
